numbat/modules/math/functions.nbt

```diff
@@ -36,6 +36,7 @@ fn atanh(x: Scalar) -> Scalar
 ## Others
 
 fn gamma(x: Scalar) -> Scalar
+fn quadratic_equation(a: Scalar, b: Scalar, c: Scalar) -> Scalar
```

Comment:
```suggestion
fn quadratic_equation(a: Scalar, b: Scalar, c: Scalar) -> (Scalar, Scalar)
```
I thought this should be `(Scalar, Scalar)`, but that gave me parsing errors when trying to run tests.

Reply:
Numbat doesn't support tuples at the moment.